modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts

```diff
@@ -487,6 +487,10 @@ declare module Liferay {
 		export function getLanguageId(): Language.Locale;
 	}
 
+	namespace PropsValues {
```

Comment:
Can we also sort this file alphabetically? I suspect Brian will bring that up sooner or later, you can just add it as an additional commit.

Reply:
Done in https://github.com/liferay-frontend/liferay-portal/pull/2115/commits/7ef3d43c2c1410acec521b9e3f2b7bd047db824e